docs/Users_Guide/matthewjetstream.rst

```diff
@@ -192,68 +211,101 @@ When the installation is complete, you can verify that the Docker command line t
+the actual weather data that was recorded during Hurricane Matthew.
+We download that data by pulling a Docker volume that holds it,
+and then referencing that volume when we run the METplus Docker container.
+The commands to pull and create the volume are::
+
+    docker pull ncar/iwrf:data-matthew-input-obs.docker
+    docker create --name ${OBS_DATA_VOL} ncar/iwrf:data-matthew-input-obs.docker
 
 Get the Geographic Data
 -----------------------
 
-To run I-WRF on the Hurricane Matthew data set, you need a copy of the
+To run WRF on the Hurricane Matthew data set, you need a copy of the
 geographic data representing the terrain in the area of the simulation.
 These commands download an archive file containing that data,
-uncompress the archive into a folder named "WPS_GEOG", and delete the archive file.
+uncompress the archive into a folder named "WPS_GEOG" in your home directory, and delete the archive file.
 They take several minutes to complete::
 
     wget https://www2.mmm.ucar.edu/wrf/src/wps_files/geog_high_res_mandatory.tar.gz
     tar -xzf geog_high_res_mandatory.tar.gz
     rm geog_high_res_mandatory.tar.gz
 
-Create the Run Folder
----------------------
+Create the WRF and METplus Run Folders
+--------------------------------------
 
 The simulation is performed using a script that must first be downloaded.
 The script expects to run in a folder where it can download data files and create result files.
-The instructions in this exercise create that folder in the user's home directory and name it "matthew".
+The instructions in this exercise create a folder (named "wrf") under the user's home directory,
+and a sub-folder within "wrf" to hold the output of this simulation.
+The subfolder is named "20161006_00", which is the beginning date and time of the simulatition.
 The simulation script is called "run.sh".
-The following commands create the empty folder and download the script into it,
-then change its permissions so it can be run::
+Similarly, a run folder named "metplus" must be created for the METplus process to use.
+The following commands create the empty folders and download the script 
+and change its permissions so it can be run::
+
+    mkdir -p ${WRF_DIR}
+    curl --location https://bit.ly/3xzm9z6 > ${WRF_DIR}/run.sh
+    chmod 775 ${WRF_DIR}/run.sh
+    mkdir -p ${METPLUS_DIR}
+
+Download Configuration Files
+----------------------------
+
+Both WRF and METplus require some configuration files to direct their behavior,
+and those are downloaded from the I-WRF GitHub repository.
+Some of those configuration files must also be copied into run folders.
+These commands perform the necessary operations::
 
-    mkdir matthew
-    curl --location https://bit.ly/3KoBtRK > matthew/run.sh
-    chmod 775 matthew/run.sh
+    git clone https://github.com/NCAR/i-wrf ${WORKING_DIR}/i-wrf
+    cp ${WRF_CONFIG_DIR}/vars_io.txt ${WRF_DIR}
+    curl --location https://bit.ly/4eKpb47 > ${WRF_DIR}/namelist.input.template
```

Comment:
Isn't this namelist file in the i-wrf GitHub repository? Should it be obtained from there instead of downloading it?

Reply:
Unfortunately, the file I am using is not quite the same as the one from GitHub.  When using the GitHub version, I got parsing errors for the lines in the @time_control section that start with "io_form_".  In the GitHub version, those lines all end with "= 2,", but that trailing comma causes problems in the test case so I had to create a version with it removed.

It's only a guess, but it may be because the run script I got from Bennett replaces some of the earlier @time_control settings with new ones.  The GitHub version of lines like "start_year" have two values in them (i.e. "2016, 2019").  The overwritten versions only have one value (i.e. "2016,").  That change on 11 lines seems to be the only difference.

I don't know how else that namelist file is being used or if removing the trailing commas would break that other usage.  Is the second column of date/time values actually needed in another usage?  Maybe we should plan to have more than one of these configuration files so that demos with different needs can each store their files in GitHub.

Actually, a fair bit of the run script is spent tweaking the namelist file contents and it might be better to just save the tweaked version in GitHub and simplify the run script.  I think that we should probably discuss these questions at a meeting before I make these changes.

Reply:
We could also discuss this with the group, but in my opinion we should include the namelist file that is used for the Hurricane Matthew use case in the GitHub repo and skip the logic in the run script that replaces values in it. Not only are there errors that are resulting in this, but it hides the details from the user who may review the namelist file we provide and become confused as to why the actual namelist file that is used to run WRF differs. We should provide a correct version of all of the configuration files that do not require any modification to run.